moped-editor/src/views/projects/projectView/CommentInputQuill.js

```diff
@@ -68,6 +68,8 @@ const CommentInputQuill = ({
               </div>
             )}
             <ProjectSaveButton
+              // disable save button if no text after removing html tags
+              disabled={!noteText.replace(/(<([^>]+)>)/gi, "")}
```

Comment:
I took a brief look at the [Quill docs](https://github.com/zenoamaro/react-quill#methods), and, since we have a ref for the autofocus behavior already, there is the option to call the `getEditor()` method and then `getLength()` to check the editor content length. It [always returns at least 1](https://github.com/quilljs/quill/issues/1105) though so that would have to be considered. 🤷

Reply:
interesting find @mddilley! @tillyw i defer to you if you want to make use of the `getLength()` method or use Frank's modified regex

Reply:
thanks, y'all! I'm going to go with @frankhereford's modified regex for simplicity, but @mddilley thanks for pointing out those methods of the Quill component!